py/desispec/scripts/zproc.py

```diff
@@ -53,23 +54,34 @@ def parse(options=None):
     """
     parser = argparse.ArgumentParser(usage="{prog} [options]")
 
+    parser.add_argument("-g", "--groupname", type=str,
+                        help="Redshift grouping type: cumulative, perexp, pernight, healpix")
+    parser.add_argument("--expfiles", nargs='*',
```

Comment:
Even if it isn't our standard usage for tile-based redshifts, `--expfiles` can be used instead of `--nights` and `--expids` to specify the NIGHT/EXPIDs to use in a file instead of on the command line.  In that case, NIGHT and EXPID are the only required columns.  I will add a check that the user doesn't provide both --expfiles and --nights/--expids.

Reply:
I was only referring to the location of these arguments with respect to the others in the `argparser` declarations. I just wanted to point out that it is an optional healpix-based argument and you had a section devoted to those. Happy to have it as an option, and also happy to keep it where it is with respect to others if you want it shown next to the other key arguments at the start.